citadel/indico_citadel/cli.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2021 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+import click
+
+from indico.cli.core import cli_group
+from indico.util.console import cformat
+
+from indico_citadel.backend import LiveSyncCitadelBackend
+from indico_livesync.models.agents import LiveSyncAgent
+
+
+@cli_group(name='citadel')
+def cli():
+    """Manage the Citadel plugin."""
+
+
+@cli.command()
+@click.argument('agent_id', type=int)
```

Comment:
I think we can skip this since the citadel backend is unique - simply query for an agent with that backend and fail if none exists.